### PR TITLE
Starting to add Eventin 1.10 core for CI and images

### PR DESCRIPTION
**Pull request by matzew**
As per title

### CODE REVIEW

File: config/eventing.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ config:
       openShiftVersions:
         - 4.12
         - 4.10
+    "release-v1.1-":
```
Comment (pierDipi) [inline config/eventing.yaml:21]:
```suggestion
    "release-v1.10":
```

Comment (matzew) [reply config/eventing.yaml:21]: lololol,

I hammered on the keyboar... and did not see a change. than committed, and was puzzled :-)

**Comment by matzew** [comment]
@pierDipi fixed :smile:

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**